src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-filter/TranscriptsFilter.tsx

```diff
@@ -71,98 +137,22 @@ const TranscriptsFilter = (props: Props) => {
           <div className={styles.header}>Sort by</div>
           <div className={styles.sortContent}>
             <RadioGroup
-              {...props}
               classNames={{
                 label: styles.label,
                 radio: styles.radio,
                 radioChecked: styles.radioChecked,
                 wrapper: styles.buttonWrapper
               }}
               options={radioData}
-              onChange={radioChange}
-              selectedOption={selectedRadio}
+              onChange={(selectedRadio) => onRadioChange(selectedRadio)}
```

Comment:
`onChange={(selectedRadio) => onRadioChange(selectedRadio)}` can be simplified to `onChange={onRadioChange}`. As the onChange is written currently, it's just a function that takes a value and passes it to a different function. No need to have this intermediary then.